embedding/embedlite/EmbedLiteView.h

```diff
@@ -71,6 +71,9 @@ class EmbedLiteViewListener
                                        const gfxSize& aScrollableSize) { return false; }
   // Some GL Context implementations require Platform GL context to be active and valid
   virtual bool RequestCurrentGLContext() { return false; }
+
+  // Will be always called from the compositor thread.
+  virtual void DrawUnderlay() {}
```

Comment:
Both are needed. This just cherry-picks https://github.com/nemomobile-packages/gecko-dev/pull/8

Cherry-picked also DrawWindowOverlay commits into this PR.